components/experimental/src/units/converter_factory.rs

```diff
@@ -38,6 +38,21 @@ impl From<Sign> for num_bigint::Sign {
     }
 }
 
+/// A struct that contains the sum and difference of base unit powers.
```

Comment:
nit: keep this in the method

Reply:
unfortunatley, it is used as output that is used in other places

Notice: `map: &mut LiteMap<u16, PowersInfo>,`

Reply:
I don't think this PR is worth it then. `insert_non_basic_units` is an implementation detail of `is_reciprocal`, and `PowersInfo` is an implementation detail of that. Having them both at the top level is not well encapsulated.